### DISCUSSION TITLE
????

### DISCUSSION THREAD

**Discussion by Theoyeah** [DEMOLITIONDON96/Demolition-Engine#11] [General (Friday Night Chattin')]
![image](https://user-images.githubusercontent.com/97792861/162255379-c94d668c-84b9-43c5-9353-34882d4d39f6.png)

**Comment by ThatOneSillyGuyDon** [comment]
Yeah, I’m lazy to remove that, not gonna touch the mods folder at all lol

**Reply by Theoyeah** [reply]
yee but why at the first place

**Reply by ThatOneSillyGuyDon** [reply]
Idk 🤷‍♂️

**Reply by Theoyeah** [reply]
okie

**Reply by ThatOneSillyGuyDon** [reply]
Also, please don’t make any new releases, I’ll do it, cause the code isn’t even done yet for the next update

**Reply by Theoyeah** [reply]
ok
